src/EnergyPlus/SingleDuct.cc

```diff
@@ -4493,6 +4494,25 @@ namespace SingleDuct {
             FanOp = 0;
             CalcVAVVS(SysNum, FirstHVACIteration, ZoneNodeNum, HCType, 0.0, 0.0, FanType, MassFlow, FanOp, QDelivered);
         }
+
+        //MassFlow = Node(SysInletNode).MassFlowRate;
+        // Move data to the damper outlet node
+        SysOutlet(SysNum).AirTemp = SysInlet(SysNum).AirTemp;
+        SysOutlet(SysNum).AirHumRat = SysInlet(SysNum).AirHumRat;
+        SysOutlet(SysNum).AirMassFlowRate = MassFlow;
+        SysOutlet(SysNum).AirMassFlowRateMaxAvail = SysInlet(SysNum).AirMassFlowRateMaxAvail;
+        SysOutlet(SysNum).AirMassFlowRateMinAvail = SysInlet(SysNum).AirMassFlowRateMinAvail;
+        SysOutlet(SysNum).AirEnthalpy = SysInlet(SysNum).AirEnthalpy;
```

Comment:
Set the SysOutlet variable for use in the UpdateSys() function. This fix will be revised in the next push to include the flow variables only.